Backend/templates/gitlab-ci/.example-gitlab-ci.yml

```diff
@@ -0,0 +1,71 @@
+
+include:
+  - remote: 'https://raw.githubusercontent.com/YouClap/Playbook/Backend/templates/gitlab-ci/.master-gitlab-ci.yml'
+
+variables:
+  GL_TOKEN: "insert gitlab token here"
```

Comment:
I think that this variable should stay in gitlab variables. I think that should be mandatory.

Reply:
this was moved to the team variables, since this is common to several projects